githublabels.py

```diff
@@ -41,6 +41,7 @@ def get_dpg_pog():
         "performance|improvements|performance-improvements",
         "mtype",
     ],
+    "ngt": [LABEL_COLORS["performance"], "ngt", "mtype"],
```

Comment:
@smuzaffar is this what you had in mind ?

Reply:
yes.